keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/AdminController.kt

```diff
@@ -75,12 +75,12 @@ class AdminController(
     adminService.backFillArtifactMetadata(type)
   }
 
-  @PostMapping(
-    path = ["/notifications/test"]
-  )
-  fun sendNotificationEvent(
-    @RequestBody event: NotificationEvent
-  ) {
-    adminService.sendNotificationEvent(event)
-  }
+//  @PostMapping(
```

Comment:
@emjburns can we remove it for now? I feel it's a bit confusing with the new implemention.

Reply:
Sure!